app/assets/javascripts/darkswarm/services/authentication_service.js.coffee

```diff
@@ -25,9 +25,3 @@ Darkswarm.factory "AuthenticationService", (Navigation, $modal, $location, Redir
     isActive: Navigation.isActive
 
     close: ->
-      if location.pathname in ["/", "/checkout"]
```

Comment:
I just asked Rob and he pointed at one use case this change breaks. If you visit `/register`, the page loads an empty layout and opens the modal. That is a weird behaviour, but probably comes from a time when everybody liked modals. When you close the modal there, it needs to redirect somewhere. Otherwise we have a blank page and the user is lost. Can you confirm that?

This behaviour doesn't make sense for the login modal, but they share the same code. Should we resolve this issue in a separate pull request?